test/indexer.py

```diff
@@ -0,0 +1,42 @@
+"""For convenience, load test data into Elasticsearch index "test".
+
+Unfortunately there's no easy way to index a JSON file using curl. (See
+https://stackoverflow.com/questions/23798433/json-bulk-import-to-elasticstearch.)
+So write a Python script.
+"""
+
+import csv
+import time
+
+from elasticsearch import Elasticsearch
+from elasticsearch import helpers
+
+
+INDEX_NAME = 'test'
+
+
+def init_elasticsearch():
+  # Wait for Elasticsearch to come up.
+  time.sleep(10)
```

Comment:
nit: Is there a way you can ping the server every say second to see if its up yet?

Reply:
That's unfortunate. This is definitely fine for this purpose, but if you're looking for a workaround I had a (kind of hacky) workaround on the old data explorer repo: 

https://github.com/bvprivate/data-explorer-service/blob/9cc7ffc373222d419ea196d8d2e897e064d99c63/data_index/test/in_memory_elastic.py#L21-L41

Reply:
(despite the bad name, that is actually a real ES not in-memory)